app/views/RoomsListView/hooks/useSubscriptions.test.tsx

```
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, act } from '@testing-library/react-native';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+import { useSubscriptions } from './useSubscriptions';
+import database from '../../lib/database';
```

ESLint

[error] 'database' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Missing file extension for "../../lib/database"

(import/extensions)